src/hotspot/share/gc/g1/g1GCPhaseTimes.cpp

```diff
@@ -482,14 +492,19 @@ double G1GCPhaseTimes::print_post_evacuate_collection_set(bool evacuation_failed
   debug_phase(_gc_par_phases[ClearCardTable], 1);
   debug_phase(_gc_par_phases[RecalculateUsed], 1);
   if (evacuation_failed) {
+    debug_phase(_gc_par_phases[PrepareRetainedRegions], 1);
     debug_phase(_gc_par_phases[RestoreRetainedRegions], 1);
+    debug_phase(_gc_par_phases[PrepareChunks], 2);
+    debug_phase(_gc_par_phases[RemoveSelfForwardsInChunks], 2);
+    debug_phase(_gc_par_phases[SyncMarkedWordInRetainedRegions], 1);
   }
 
   trace_phase(_gc_par_phases[RedirtyCards]);
   debug_time("Post Evacuate Cleanup 2", _cur_post_evacuate_cleanup_2_time_ms);
   if (evacuation_failed) {
     debug_phase(_gc_par_phases[RecalculateUsed], 1);
     debug_phase(_gc_par_phases[RestorePreservedMarks], 1);
+    debug_phase(_gc_par_phases[VerifyAfterSelfForwardingPtrRemoval], 1);
```

Comment:
That should probably only be printed when the respective verification is enabled, i.e. when this task is actually run.